test/spec/services/ressourceService.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+describe('ResultatService', function () {
+    describe('applyYearlyRessource', function() {
+        var service, individu, dateDeValeur, ressource;
```

Comment:
Put each `var` declaration on its own line. Less code churn and more readability.